roles/openshift_serviceaccounts/tasks/main.yml

```diff
@@ -0,0 +1,23 @@
+- name: Create service account configs
+  template:
+    src: serviceaccount.j2
+    dest: "/tmp/{{ item }}-serviceaccount.yaml"
+  with_items: accounts
+
+- name: Create {{ item }} service account
+  command: >
+    {{ openshift.common.client_binary }} create -f "/tmp/{{ item }}-serviceaccount.yaml"
+  with_items: accounts
+
+- name: Get current security context constraints
+  shell: oc get scc privileged -o yaml > /tmp/scc.yaml
+
+- name: Add security context constraint for {{ item }}
+  lineinfile: dest=/tmp/scc.yaml line="- system:serviceaccount:default:{{ item }}"
+  with_items: accounts
```

Comment:
Perhaps add insertafter to ensure we're after `users:` line? Adding to the end of the file today is safe today but may not be in future.

```
  lineinfile:
    dest: /tmp/scc.yaml
    line: "- system:serviceaccount:default:{{ item }}"
    insertafter: "^users:$"
```